components/style/selector_matching.rs

```diff
@@ -165,39 +165,35 @@ impl Stylist {
         // Take apart the StyleRule into individual Rules and insert
         // them into the SelectorMap of that priority.
         macro_rules! append(
-            ($style_rule: ident, $priority: ident, $importance: expr, $count: expr) => {
```

Comment:
Could you add a comment here about why we can't rip this out if we know there are no declarations?